stdlib/pushbullet/pushbullet.flux

```diff
@@ -6,11 +7,42 @@ import "json"
 
 option defaultURL = "https://api.pushbullet.com/v2/pushes"
 
-// `pushData` sends a push notification using PushBullet's APIs.
-// `url` - string - URL of the PushBullet endpoint. Defaults to: "https://api.pushbullet.com/v2/pushes".
-// `token` - string - the api token string.  Defaults to: "".
-// `data` - object - The data to send to the endpoint. It will be encoded in JSON and sent to PushBullet's endpoint.
+// The pushbullet.pushData() function sends a push notification to the Pushbullet API.
```

Comment:
The docs spec states the convention here is to start the sentence with the name of the function i.e. 

```
pushData sends a push notification to the Pushbullet API.
```

Can you update the first line to follow that convention?